content/projects/gsoc/2022/projects/jenkinsfile-runner-action-for-github-actions.adoc

```diff
@@ -102,6 +102,7 @@ link:/chat/#continuous-delivery-foundation[Explanations about the CDF Slack] (th
 === Project Links
 Here come some useful links:
 
+* link:https://docs.google.com/presentation/d/1t2vuNn1NFpDusnw0m4vdFw6WBQMeU6kccv_K1v2L6R0/edit#slide=id.g13dcaed2105_0_8/[Midterm Presentation at 8th page]
```

Comment:
```suggestion
* link:https://docs.google.com/presentation/d/1t2vuNn1NFpDusnw0m4vdFw6WBQMeU6kccv_K1v2L6R0/edit#slide=id.g13dcaed2105_0_8/[Midterm Presentation Slides]
```

Reply:
When I previewed the deployment, I wasn't directed to page 8 of the slides right away, but to the first page

Reply:
That's weird. When I tested locally, it could jump to 8th page. Never mind. Just use the current one.